bench/perf_run_py/perf_run.py

```diff
@@ -320,13 +321,17 @@ def run_perf_tests(config: PerfConfig,
             if args.verbose:
                 print("Batch test {}: Arguments: {}, Operations: {}".
                       format(index,  content["arguments"], content["operations"]))
-                perf_stats = PerfStatCollection(content["operations"])
-                if not args.reuse:
-                    run_test_wrapper(config=config, index=index, arguments=content["arguments"])
-                reported_stats += process_results(config, perf_stats, index=index)
+            perf_stats = PerfStatCollection(content["operations"])
+            if (config.improved_accuracy):
```

Comment:
The `if (config.improved_accuracy)` clause here is outside of the `if not args.reuse` clause, but a few lines down in line 327 it's inside the `if not args.reuse`. Could we make this consistent?

Reply:
Good catch! Fixed in [22beadc](https://github.com/wiredtiger/wiredtiger/pull/8506/commits/22beadce7de73bf542693031682f0ba91c1cfaa0)